tests/fft_c2c.py

```diff
@@ -0,0 +1,290 @@
+# Copyright 2021 NVIDIA Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import warnings
+
+import numpy as np
+
+import cunumeric as num
+
+
+def allclose(A, B):
+    if B.dtype == np.float32 or B.dtype == np.complex64:
+        l2 = (A - B) * np.conj(A - B)
+        l2 = np.sqrt(np.sum(l2) / np.sum(A * np.conj(A)))
+        return l2 < 1e-6
+    else:
+        return np.allclose(A, B)
+
+
+def test_1d(N, dtype=np.float64):
+    Z = np.random.rand(N).astype(dtype) + np.random.rand(N).astype(dtype) * 1j
+    Z_num = num.array(Z)
+
+    out = np.fft.fft(Z)
```

Comment:
I guess you could use a for loop to iterate the test vectors, instead of enumerating them like this.